android/src/main/java/com/reactnativemembrane/MembraneModule.kt

```diff
@@ -90,20 +90,37 @@ class MembraneModule(reactContext: ReactApplicationContext) :
     return res
   }
 
+  private fun ReadableMap.toMap(): MutableMap<String, Any> {
```

Comment:
Do we need all of those booleans, numbers etc? Shouldn't strings suffice? If so, we have already function for that: `ReadableMap.toMetadata`